src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -115,8 +115,8 @@ void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
         resultList.getChildren().add(personListPanel.getRoot());
 
-        resultDisplay = new ResultDisplay();
-        responseDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
+        responseDisplay = new ResultDisplay();
```

Comment:
Change the name of the class as well? i.e. ResultDisplay to ResponseDisplay

Reply:
Got it, Ill make more changes to follow the new name